src/index.html

```diff
@@ -3,11 +3,311 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="shortcut icon"
+      href="images/favicon.ico"
+      type="image/x-icon"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
```

Comment:
header should cover the whole viewport at least in tablet and laptop resolutions
![cover full screen on laptop and tablet](https://user-images.githubusercontent.com/39308717/163113740-17b481d2-002c-4ef9-9c77-dec5406eacc3.png)

Reply:
Done!